src/style.css

```
@@ -1 +1,140 @@
-/* empty source */
+/* #region page */
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+@font-face {
+  font-family: "Roboto", sans-serif;
+}
+
+/* #endregion */
+/* #region card */
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border: solid #f3f3f3 1px;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.card__item-macspecs {
+  padding-top: 33px;
+  width: 160px;
+  height: 134px;
+  padding-left: calc(50% - 160px / 2);
+}
+
+.card__item-macspecs-text {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  padding: 17px;
+  padding-bottom: 4px;
+}
+
+.card__item-prdcode {
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  padding: 0.01px 17px;
+  color: #616070;
+}
+/* #endregion */
+/* #region stars */
+.card__item-stars {
+  display: flex;
+  list-style: none;
+  padding: 16px;
+}
+
+.stars {
+  display: flex;
+  list-style: none;
+  padding: 16px;
+}
+  
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: auto;
+}
+  
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+  
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars__star-rev {
+  display: flex;
+  align-items: flex-end;
+  text-align: right;
+  margin-left: 17px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+/* #endregion */
+/* #region price */
+.card__item-price {
+  display: flex;
+  justify-content: space-between;
+  padding: 16px;
+  padding-top: 6px;
+  font-family: "Roboto", sans-serif;
+}
+
+.card__item-price-text {

```

`item` doesn`t provide any meaning, remove it throughout your code
```suggestion
.card__price-text {

```